docs/dev-tools/zombienet/overview.md

```diff
@@ -170,11 +176,11 @@ It's important to note that each provider has specific requirements and associat
     !!! note
         When network operations cease—either by halting a running spawn with Ctrl+C or upon completion of the test—Zombienet automatically removes all associated pods, including those for Prometheus, Tempo, and Grafana.
 
-### Native
+### Local
 
 === "Requirements"
```

Comment:
For this providers section, they do want it broken out into sections instead of these tabs. Please rework/reformat, commit changes, and request a new review.